can not run

```
$> scoop search zoc
panic: runtime error: index out of range [1] with length 1

goroutine 7 [running]:
main.matchingManifests(0xc0001800f0, 0x28, 0xc000010098, 0x3, 0x0, 0x0, 0x0)
        /home/runner/work/scoop-search/scoop-search/main.go:133 +0x106b
main.main.func1(0xc00000c420, 0x1c, 0xc000010098, 0x3, 0x0, 0xc000052280, 0xc000010170, 0x862f00, 0xc000122000)
        /home/runner/work/scoop-search/scoop-search/main.go:70 +0x185
created by main.main
        /home/runner/work/scoop-search/scoop-search/main.go:63 +0x253
```
version:1.1.0
OS:win11

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -16,7 +16,7 @@ jobs:
       - name: Set up Go 1.x
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.13
+          go-version: ^1.17
         id: go
 
       - uses: actions/checkout@v2

--- a/main.go
+++ b/main.go
@@ -132,7 +132,10 @@ func matchingManifests(path string, term string) (res []match) {
 					case fastjson.TypeArray:
 						// check only first two, the rest are command flags
 						stringArray := stringOrArray.GetArray()
-						bins = append(bins, string(stringArray[0].GetStringBytes()), string(stringArray[1].GetStringBytes()))
+						bins = append(bins, string(stringArray[0].GetStringBytes()))
+						if len(stringArray) > 1 {
+							bins = append(bins, string(stringArray[1].GetStringBytes()))
+						}
 					default:
 						log.Fatalln(badManifestErrMsg)
 					}

--- a/main.go
+++ b/main.go
@@ -47,7 +47,7 @@ func main() {
 	bucketsPath := scoopHome() + "\\buckets"
 
 	// get specific buckets
-	buckets, err := ioutil.ReadDir(bucketsPath)
+	buckets, err := os.ReadDir(bucketsPath)
 	checkWith(err, "Scoop folder does not exist")
 
 	// start workers that will find matching manifests
@@ -59,8 +59,12 @@ func main() {
 	var wg sync.WaitGroup
 
 	for _, bucket := range buckets {
+		if !bucket.IsDir() {
+			continue
+		}
+
 		wg.Add(1)
-		go func(file os.FileInfo) {
+		go func(file os.DirEntry) {
 			// check if $bucketName/bucket exists, if not use $bucketName
 			bucketPath := bucketsPath + "\\" + file.Name()
 			if f, err := os.Stat(bucketPath + "\\bucket"); !os.IsNotExist(err) && f.IsDir() {

--- a/main.go
+++ b/main.go
@@ -2,7 +2,6 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
@@ -88,7 +87,7 @@ func main() {
 
 func matchingManifests(path string, term string) (res []match) {
 	term = strings.ToLower(term)
-	files, err := ioutil.ReadDir(path)
+	files, err := os.ReadDir(path)
 	check(err)
 
 	var parser fastjson.Parser
@@ -102,7 +101,7 @@ func matchingManifests(path string, term string) (res []match) {
 		}
 
 		// parse relevant data from manifest
-		raw, err := ioutil.ReadFile(path + "\\" + name)
+		raw, err := os.ReadFile(path + "\\" + name)
 		check(err)
 		result, _ := parser.ParseBytes(raw)
 

